catalog/entry_catalog.go

```diff
@@ -5,8 +5,12 @@ import (
 	"crypto"
 	_ "crypto/sha256"
 	"fmt"
+	"time"
+
+	"github.com/treeverse/lakefs/actions"
 
```

Comment:
dd

Reply:
but I'm using the constants for the event type from actions